app/services/query-service.js

```diff
@@ -80,23 +80,52 @@ const runFetchAnswerQuery = async ({ query, chatHistory, summariesMode, embeddin
   } catch (error) {
     trackFetchResponseFailed({
       errorMessage: error.message,
-      requestQuery: query
+      requestQuery: query,
+      retryCount
     })
     return {
       response: { answer: 'This tool cannot answer that kind of question, ask something about Defra funding instead', hallucinated: true }
     }
   }
 }
 
-/**
```

Comment:
Any reason for removing the jsdocs? Besides them looking ugly - it does provide type hinting